plugins/services/src/js/reducers/JSONPodReducers.js

```diff
@@ -0,0 +1,23 @@
+import {JSONReducer as constraints} from './serviceForm/Constraints';
```

Comment:
Let's rename this file to `JSONMultiContainerReducers` to be consistent with the Volumes and Network reducers